docs/publish/centralized-deployment.md

```diff
@@ -13,7 +13,7 @@ The Office 365 admin center makes it easy for an administrator to deploy Office
 The Office 365 admin center currently supports the following scenarios:
 
 - Centralized Deployment of new and updated add-ins to individuals, groups, or an organization.
-- Deployment to multiple platforms, including Windows, Mac, and on the web.
+- Deployment to multiple platforms, including Windows, Mac, mobile, and on the web.
```

Comment:
Should we call out Tabs like iPad separately. Or can we look to the usage of term "mobile devices" to encompass Tablets